README.md

```diff
@@ -75,9 +80,9 @@ Select 1:
 Identify the three places CSS can go, and rank them in terms of specificity:
 
 ```text
-1.
-2.
-3.
+1.inline
+2.<head>
+3.stylesheet
 ```
 
```

Comment:
Correct but just to expand on answers 2 & 3 

2 `<style>` tags within the `<head>`
3  external stylesheets